images/scripts/services.setup

```diff
@@ -116,27 +116,21 @@ chmod 755 /root/run-samba-domain
 #
 #############################
 
-# unfortunately the setup only works on RHEL/CentOS 7: https://github.com/candlepin/ansible-role-candlepin/pull/12
-# so we have to run that in a CentOS 7 container
-
-podman run --name=candlepin --uts=host --publish=8443:8443 --privileged --detach quay.io/centos/centos:centos7 /sbin/init
+podman run --name=candlepin --uts=host --publish=8443:8443 --privileged --detach -t quay.io/fedora/fedora:35 sh
```

Comment:
How about `:latest` instead of `:35` so that this keeps itself up to date?

Reply:
Done